### ISSUE TITLE
No more position for HmIP-HDM2 (cover) 

### ISSUE THREAD

**Issue by zngguvnf** [SukramJ/hahomematic#1116]
### I agree to the following

- [X] I have read the [documentation](https://github.com/danielperna84/custom_homematic#custom_homematic)
- [X] I am aware of the latest [release notes](https://github.com/danielperna84/custom_homematic/releases)
- [X] The Backend (CCU/Homegear/...) is working as expected and connected devices are controllable by its own UI.
- [X] I am running the latest version of the custom_component (and Home Assistant)

### The problem

Since the last update (unfortunately I updated homeassistant, hahomematic and raspberrymatic at the same time) 
I can no longer the the current status from my blinds (HmIP-HDM2) in HomeAssistant.
However, the control from HomeAssistant still works.
In RaspberryMatic, values for the position of the blinds are still displayed (between 0.0 (closed) and 1.0 (open).

Current attributes from cover (Developer tools -> Entity -> Attributes:
```
interface_id: raspberrymatic-HmIP-RF 
address: 0000C882BE01E3:1 
model: HmIP-HDM2 
entity_type: custom 
value_state: restored 
friendly_name: plissee_flur_rechts 
supported_features: 11
```

State: `unknown`.

Do you have any idea what's going wrong here? I can't find anything related in the logs.

### What version of HomematicIP (local) has the issue?

1.39.2

### What was the last working version of HomematicIP (local)?

unknown

### What type of installation are you running?

Home Assistant OS

### What type of installation are you running for your homematic backend?

RaspberryMatic Standalone

### Which version of your homematic backend are you running?

3.69.7.20230626

### What hardware are you running for your system?

Proxmox

### Which config details do you use

- [X] Username and password
- [ ] TLS
- [ ] callback data ([see](https://github.com/danielperna84/custom_homematic#callback_host-and-callback_port))

### Which interfaces do you use?

- [X] Homematic IP
- [ ] Homematic (Bidcos-RF)
- [ ] Groups (Heating-Group)
- [ ] BidCos-Wired (HM-Wired)

**Comment by SukramJ** [comment]
`value_state: restored ` means that HA was not able to fetch data from CCU or has not received data since last restart.
I don't own a HmIP-HDM2 so i cannot test it by my self.

Enable debug logging for this integration, change values for your device in CCU.
Check the value_state of the cover entity again. Should be valid now.
If not, download the full log from HA and add it here as file.

Try to remember the last working version of integration. There has been no change for cover related code for 3 month!

**Comment by SukramJ** [comment]
Additionally search for more entities with `value_state: restored`.
In that case there might be another root cause.

**Comment by zngguvnf** [comment]
Thanks for your fast reply. 

Issue was fixed with homeassistant 2023.7.3
